thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/dashboard/ThirdEyeDashboardApplication.java

```diff
@@ -139,36 +114,11 @@ public void run(ThirdEyeDashboardConfiguration config, Environment env)
     final JerseyEnvironment jersey = env.jersey();
     injector = Guice.createInjector(new ThirdEyeDashboardModule(config, env, DAO_REGISTRY));
     Stream.of(
-        DetectionConfigurationResource.class,
-        DatasetAutoOnboardResource.class,
-        DashboardResource.class,
-        CacheResource.class,
         AnomalyResource.class,
-        EntityManagerResource.class,
-        MetricConfigResource.class,
-        DatasetConfigResource.class,
-        AdminResource.class,
-        SummaryResource.class,
-        ThirdEyeResource.class,
-        DataResource.class,
-        AnomaliesResource.class,
-        EntityMappingResource.class,
-        OnboardDatasetMetricResource.class,
-        AutoOnboardResource.class,
-        ConfigResource.class,
-        CustomizedEventResource.class,
-        AnomalyFlattenResource.class,
-        UserDashboardResource.class,
-        ApplicationResource.class,
-        DetectionResource.class,
-        DetectionAlertResource.class,
-        YamlResource.class,
-        SqlDataSourceResource.class,
-        AlertResource.class,
-        RootCauseTemplateResource.class,
-        RootCauseSessionResource.class,
         RootCauseMetricResource.class,
-        AnomalySearchResource.class
```

Comment:
it seems this one - `AnomalySearchResource`  is not added to the RootResource.

Reply:
@jihaozh 
correct. That's because it didn't seem to exist prior to my changes (before #5792 ). There was no usage of the `AnomalySearch` class and it wasn't registered in jersey. Hence, I didn't add it. Let me know and I can add it back.

Reply:
Yes. I figured out that this one was added in the last couple of days. So the problem for this PR is that since it refactors Resource classes, it needs to be merged quickly else it has to be updated with every incoming resource PR.